.mocharc.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  require: [
```

Comment:
Can we add a few more settings to the `mocha` configuration?

```
  opts: false,
  slow: 500,
  timeout: 5000,

  // Check for global variable leaks
  "check-leaks": true,
```

`opts:false` tells Mocha not to look for the `mocha.opts` file.
`slow` and `timeout` both warn if tests take too long. I'm not sure what good values for those are for this library, but these are the values used in `ripple-lib`. And then checking for global variable leaks is just a good thing to do I feel like.